1.Hello-World/Hello-World/ViewController.m

```diff
@@ -88,9 +88,9 @@ - (void)doConnect
  */
 - (void)doPublish
 {
-    _publisher =
-    [[OTPublisher alloc] initWithDelegate:self
-                                     name:[[UIDevice currentDevice] name]];
+    OTPublisherSettings *setting = [[OTPublisherSettings alloc] init];
```

Comment:
I recommend changing the name of this property to `_settings` (with an s) or maybe `settings`.